test/legacy-integration/notifications/push-notification.spec.ts

```diff
@@ -0,0 +1,28 @@
+import {
```

Comment:
I'm wondering how much we should add to the `legacy-integration`. We're aiming at trimming this folder down. @vindard  would have more context